odkx-src/sync-endpoint-cloud-setup.rst

```diff
@@ -344,30 +350,51 @@ Connecting to your virtual machine
 
     $ sudo ./script_to_run.sh
 
-  The script will ask you for the server's domain and an administration email address to configure https on the server.
+  The script will ask you for the server's domain and some questions (as shown in the picture below) along with an administration email address to configure https on the server.
+
+  .. image:: /img/setup-azure/azure-connecting-to-virtual-machine1.png
+    :width: 600
 
   After gathering this data the script will begin the install and you should see a bunch of statements executing in your console. Wait approximately 5-10 minutes for the installation to complete.
 
-  .. image:: /img/setup-azure/azure8.png
+  .. image:: /img/setup-azure/azure9.png
     :width: 600
 
   Once all the services have been created, we need to check if all the services are running properly with the command:
 
   .. code-block:: console
 
-    $ docker stack ls
+    $ sudo docker stack ls
 
-  If there are 8 (or 7 without https) services running under the name
-  `syncldap`, everything is running properly.
+  To see all of the Docker processes/containers that are actively running, use the following command:
+
+  .. code-block:: console
+
+    $ sudo docker ps
+
+  .. image:: /img/setup-azure/azure-connecting-to-virtual-machine2.png
+    :width: 600
+
+  If there are 9 (or 8 without https) services running under the name
+  `syncldap`, everything is running properly. The following command shows all (both stopped and running) containers/processes:
+
+  .. code-block:: console
+
+    $ sudo docker ps -a
+
+  .. image:: /img/setup-azure/azure-connecting-to-virtual-machine3.png
+    :width: 600
+
+  docker ps command shows the 7 actively running services, while docker ps -a shows all the stopped and running services. Hence, 7 active, 1 stopped and https (total 9) services run on the server as shown by docker stack ls command.
```

Comment:
Docker `ps -a` can be confusing to folks new to docker, and is especially weird with docker swarm. Even in the example screenshot you added there are actually 11 containers, not 9! `ps -a` shows all containers, even the ones that are stopped, and with the way swarm works some extra containers may start and stop right as the system is coming online. There won't be a consistent number of containers listed with `ps -a` depending on the specific speed of the computer it's running on and the order things start up. I think it might be better to say there should be 9 services (or 7 if not using https) shown by `docker stack ls` , and 7 (or 6 if not using https) actively running shown by `docker ps`. If you do want to have the part about `docker ps -a`, you will probably need to add some more explanation of what it's doing and how to interpret the output!

Reply:
Yeah, I think removing ps -a would be better to avoid any confusion for the new users.